odoo/addons/base/models/res_partner.py

```diff
@@ -606,6 +625,51 @@ def _update_fields_values(self, fields):
                 values[fname] = self[fname]
         return values
 
+    @api.model
+    def _get_eu_prefixed_countries(self):
+        return (
+            self.env.ref('base.europe').country_ids
+            + self.env.ref('base.ch')
+            + self.env.ref('base.no')
+            + self.env.ref('base.uk')
+            + self.env.ref('base.sm')
+        )
```

Comment:
Should this not be a country group? (then using `country_id.country_group_codes`)
It would avoid many calls to `ref`, as well as avoid relying on xmlids